conmon-rs/common/proto/conmon.capnp

```diff
@@ -23,4 +27,20 @@ interface Conmon {
     }
 
     createContainer @1 (request: CreateContainerRequest) -> (response: CreateContainerResponse);
-}
+
```

Comment:
i think these changes should go in the second commit

Reply:
hmm. the second commit is just the regeneration of the files.